.github/workflows/pr_agent_template.yml

```diff
@@ -0,0 +1,29 @@
+name: PR Agent Template
+
+on:
+  workflow_call:
+    inputs:
+      openai_key:
+        required: true
+        type: string
+      github_token:
+        required: true
+        type: string
+
+jobs:
+  pr_agent_job:
+    if: github.event.pull_request.state == 'open'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: Codium-ai/pr-agent@main
+        env:
+          pr_reviewer.enable_review_labels_security: false
+          pr_reviewer.enable_review_labels_effort: false
+          OPENAI_KEY: ${{ inputs.openai_key }}
```

Comment:
❓ 

Given such secrets are shared across the Github organisation for our repositories, is it necessary to template their value?